cpupower_gui/helper.py

```diff
@@ -209,6 +219,11 @@ def set_cpu_max_freq(cpu, freq):
 
 def get_cpu_frequencies(cpu):
     """Return frequencies for cpu"""
-    fmin, fmax = read_freqs(cpu)
-    hmin, hmax = read_freq_lims(cpu)
+    fmin = 0
+    fmax = 0
+    hmin = 0
+    hmax = 0
+    if is_online(cpu):
```

Comment:
This will not be needed if `is_online` is used in `read_freqs` and `read_freq_lims` internally.